lib/superbot/web.rb

```diff
@@ -28,9 +29,30 @@ def initialize
       end
 
       @sinatra.post "/__superbot/v1/convert" do
-        body = JSON.parse(request.body.read)
+        converted_body = Superbot::Capybara::Convert.call(request.body.read)
+        begin
+          gem 'superbot-capybara'
 
-        "visit 'http://example.com'\n#{body.inspect}"
+          k = Kommando.new "sb-capybara"
```

Comment:
It would be nicer to extract this to a multi purpose class already at this point (sb-robot etc)